plugins/compilers/raspc-rasp/src/main/jflex/lexer.jflex

```diff
@@ -85,6 +85,8 @@ number = "-"?[0-9]+
 identifier = [a-zA-Z][a-zA-Z0-9]*
 label = {identifier}[":"]
 operator_constant = "="
+org=[\r|\n|\r\n]*"org"
```

Comment:
why there is the prefix `[\r|\n|\r\n]` ?? It is wrong.

Reply:
this was to enable empty lines before the directive, but probably should be handled in the grammar file

Reply:
or should it be OK just to force user not to add any empty lines?

Reply:
Well I think the user should be able to insert empty lines... And the grammar file theoretically does not have to mitigate this, because every sensible input is per-line. When you removed these `\r|\n...` it does not work out of the box? It should.